app/models/pipeline_run.rb

```diff
@@ -291,6 +297,29 @@ def invalid_family_call?(tcnt)
     false
   end
 
+  def db_load_amr_counts
+    # amr_s3_path = "#{postprocess_output_s3_path}/#{AMR_OUTPUT_NAME}"
+    amr_test_s3_path = "s3://idseq-database/test/AMR/__fullgenes__ARGannot_r2__results.txt"
+    amr_downloaded_path = PipelineRun.download_file(amr_test_s3_path, local_txt_path)
```

Comment:
I think you'll want to change local_txt_path to something pipeline_run-specific so that there are no clashes between runs. E.g. "local_txt_path = app/temp/amr_results/#{id}"

Reply:
Oh, just saw you did this.